app/controllers/admin/videos_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Admin::VideosController < AdminsController
+  def new
+    @video = Video.new
+  end
+
+  def create
+    @video = Video.new(video_params)
+    category = Category.find(params[:video][:categories]) if params[:video][:categories]
+    @video.categories << category
```

Comment:
You have a many-to-many setup but this code seems like you are only putting a video in a single category? That should be a one-to-many.